build.gradle

```diff
@@ -1,4 +1,4 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+// Top-level build file where you can add configuration options commogit n to all sub-projects/modules.
```

Comment:
Here I think you inadvertently modify `common` by `commogit n`. I would put the comment back as it was before.